.gitignore

```diff
@@ -24,4 +24,7 @@ pathfinder-var.env
 .vscode/
 
 # mdbook compilation
-**/book/
\ No newline at end of file
+**/book/
+
+# Intellij IDE generated files
+.idea
```

Comment:
Pretty sure that the Pathfinder guys will disapprove this. Note that you can set up a global `.gitignore` file in your home directory to avoid this.

Reply:
cool